src/Articles.jsx

```diff
@@ -0,0 +1,45 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+import Col from "react-bootstrap/Col";
+import Row from "react-bootstrap/Row";
+import Card from "react-bootstrap/Card";
+import './ArticleCard.css'
+
+const Articles = () => {
+
+    const [articles, setArticles] = useState([]);
+        useEffect(() => {
+            axios.get("https://news-that-matters.onrender.com/api/articles/")
```

Comment:
This axios logic should be packed away into a separate utils file called `api.js` 👍